lib/gems/pending/metadata/util/md5deep.rb

```diff
@@ -244,7 +242,7 @@ def to_xml
 
   # Mount VM Image to a real drive letter
   #  mountNative, startPath = false, "M:/WINDOWS/system32/mui"
-  startPath = "c:/windows/system32"
+  #  startPath = "c:/windows/system32"
```

Comment:
Looks like this `startPath` comment could be merged with the one above.  Also makes me wonder if there is a reason for these comments.